cmd/kubeadm/app/cmd/init.go

```diff
@@ -221,15 +222,18 @@ func (i *Init) Validate(cmd *cobra.Command) error {
 // Run executes master node provisioning, including certificates, needed static pod manifests, etc.
 func (i *Init) Run(out io.Writer) error {
 
-	// PHASE 1: Generate certificates
-	err := cmdphases.CreatePKIAssets(i.cfg)
+	k8sVersion, err := version.ParseSemantic(i.cfg.KubernetesVersion)
 	if err != nil {
+		return fmt.Errorf("couldn't parse kubernetes version %q: %v", i.cfg.KubernetesVersion, err)
+	}
+
+	// PHASE 1: Generate certificates
+	if err := cmdphases.CreatePKIAssets(i.cfg); err != nil {
```

Comment:
Much of the rest of the file uses this style so I think that it is best to at least be consistent here. This is a little out of step of much other go code in the k8s code base so it might be worth rationalizing at some point.